pyModeS/decoder/bds/bds09.py

```diff
@@ -66,6 +74,7 @@ def airborne_velocity(msg):
 
         tag = 'GS'
         trk_or_hdg = round(trk, 2)
+        dir_type = 'gnd_trk'
```

Comment:
'true_north" would be better here

```suggestion
        dir_type = 'true_north'
```